setup.cfg

```diff
@@ -39,8 +39,7 @@ scikit-quant =
 cma = 
     cma>=3.0.0
 qiskit =
-    qiskit>=0.28, <0.34;python_version<'3.10'
-    qiskit>=0.34, <0.37;python_version>='3.10'
+    qiskit==0.38
```

Comment:
I think we should change it to `orquestra-qiskit` 🤔 
This would make it easier to avoid future issues with making all the libs compatible.

Reply:
Maybe this would be a good idea. Since qiskit is an optional install it wouldn't necesarily depend on `orquestra-qiskit`. @dexter2206 what do you think?